ansible/setup.yml

```diff
@@ -98,9 +98,15 @@
         - ./configure
         - make
         - make install
-        - ldconfig
       args:
         chdir: "/usr/local/src/libsodium-{{ libsodium_version }}"
         creates: "/usr/local/lib/libsodium.so.{{ libsodium_module_version }}"
       tags:
         - libsodium
+      register: installed
+
+    - name: Run ldconfig
+      command: ldconfig
+      tags:
+        - libsodium
+      when: installed.rc #workaround for ansible-lint [ANSIBLE0012] Commands should not change things if nothing needs doing ansible/setup.yml:114
```

Comment:
not sure this is quite right, because it's exit code of the command (0 on success), so I read the task as "run on failed install"

I thin it should be `when: installed.rc == 0`